src/coffeeShops/CoffeeShopsChart.tsx

```diff
@@ -49,28 +84,68 @@ function createOptions() {
         callbacks: {
           title: tooltipTitle,
         },
+        animation: {
+          duration: 300,
+        },
       },
     },
+    onClick: onClick,
+    animation: {
+      duration: 300,
+    },
   };
 }
 
 function tooltipTitle(tooltipItems: TooltipItem<"scatter">[]) {
-  return tooltipItems.map((item) => (item.raw as CoffeeShop).name).join("\n");
+  return tooltipItems.map((item) => (item.raw as DataPoint).title).join("\n");
+}
+
+function toChartCoordinates(
```

Comment:
https://www.chartjs.org/docs/latest/configuration/interactions.html#converting-events-to-data-values
This should help

Reply:
I am referring to the helpers to convert click coordinates into Chart Points. `Chart.helpers.getRelativePosition(e, chart);`

Reply:
OK, I will change it.

Reply:
Made the change on the dev branch.